src/main/webapp/script.js

```diff
@@ -53,3 +54,19 @@ function displayCalendar(calId) {
   const url = "https://calendar.google.com/calendar/embed?src=" + calId;
   document.getElementById("calendar").src = url;
 }
+
+// delete all secondary calendars created, except for the chosen one 
+function deleteUnchosenCalendars() {
+  idArray
+      .filter((id, index) => index !== currCalendarIndex)
+      .forEach(id => deleteCalendar(id));
+
+  const chosenCalendarId = idArray[currCalendarIndex];
+  idArray = [chosenCalendarId];
```

Comment:
You'll probably want to set `currCalendarIndex` to 0 here also.

Reply:
Good point, I'll make that change.